astro/components/SiteTitle.astro

```diff
@@ -0,0 +1,76 @@
+---
+import AstrolightSiteTitle from "@astrojs/starlight/components/SiteTitle.astro";
+import type { Props } from "@astrojs/starlight/props";
+
+const menuItems = [
+  { name: "Docs", href: "/introduction" },
+  { name: "Performance", href: "/performance" },
+  { name: "Playground", href: "/playground" },
+];
+
+function pathsMatch(pathA: string, pathB: string): boolean {
+  return pathB.includes(pathA);
+}
+---
+
+<div>
+  <AstrolightSiteTitle {...Astro.props} />
+
+  <div class="sl-hidden separator"></div>
+
+  <div class="sl-hidden menu-items">
+    {
+      menuItems.map((menuItem) => (
+        <a
+          href={menuItem.href}
+          aria-current={
+            pathsMatch(encodeURI(menuItem.href), Astro.url.pathname) && "page"
+          }
+        >
+          <span>{menuItem.name}</span>
+        </a>
+      ))
+    }
+  </div>
+</div>
+
+<style>
+  div {
```

Comment:
Can we use `tailwindcss` in here?

Reply:
If we can use it, I just thought they wouldn't use Tailwind anymore since it wasn't integrated with Astro.

Reply:
I don’t think it would be a good idea to integrate Tailwind yet, to maintain consistency with the Next.js page and avoid altering unwanted styles.